assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -112,6 +113,7 @@ function TextPane(props) {
         ))}
       </Section>
       <Section title={__('Text Sets', 'web-stories')} />
+      {__('Coming soon.', 'web-stories')}
```

Comment:
Should it maybe go in a `<p>` or something? Or is default styling okay here?

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/975/commits/13691992cf61a8383f29d0ed470b0644954ee90b